mass2/core/channels.py

```diff
@@ -359,15 +395,20 @@ def save_recipes(
         return steps
 
     def load_recipes(self, filename: str) -> "Channels":
+        """Return a copy of this Channels object with Recipe objects loaded from the given pickle file."""
```

Comment:
This one is prettty bad, should say "Return a copy of this Channels object with the Reciple objects loaded from the given pickle file applied to each contained Channel."